### PR TITLE
Add new app constants

### PR DESCRIPTION
**Pull request by JeffResc**
I added a bunch of app constants from my app library to help new users have more defaults available. You can view all of them [in the files section](https://github.com/JeffLIrion/python-androidtv/pull/201/files).

Hope this helps!

### CODE REVIEW

File: androidtv/constants.py [hunk 1]
```diff
@@ -268,18 +268,27 @@
 
 
 # Apps
+APP_AE_TV = 'com.aetn.aetv.watch'
+APP_APPLE_TV = 'com.apple.atve.amazon.appletv'
```
Comment (JeffLIrion) [inline androidtv/constants.py:272]: Can you please put this in alphabetical order.

File: androidtv/constants.py [hunk 2]
```diff
@@ -323,15 +346,20 @@
         APP_PLAY_VIDEOS: 'Play Videos',
         APP_PLEX: 'Plex',
         APP_PRIME_VIDEO: 'Prime Video',
+        APP_PRIME_VIDEO_2: 'Prime Video',
```
Comment (JeffLIrion) [inline androidtv/constants.py:349]: The friendly names need to be unique. Same goes for YouTube.

Comment (JeffResc) [reply androidtv/constants.py:349]: @JeffLIrion How would you like to handle when an app has two different identifiers (i.e. YouTube)? What should it be renamed as?

File: androidtv/constants.py (hunk 1 above)
Comment (JeffResc) [reply androidtv/constants.py:272]: Oops, thought I did this, my bad

File: androidtv/constants.py (hunk 2 above)
Comment (JeffLIrion) [reply androidtv/constants.py:349]: I'll leave that up to you. For YouTube, maybe "YouTube (Fire TV)". I'm not sure what the difference is between these two Amazon Prime Video apps.

Comment (JeffResc) [reply androidtv/constants.py:349]: I noticed that a few of them mention `eu`, but I'm in North America so I'm wondering if it's possibly just a regional difference

**Comment by JeffLIrion** [comment]
I'll merge this once the `APP_*` constants are alphabetized and the friendly names are unique. 

For the Prime Video app, you could call it "Prime Video (2)" if you can't come up with a better name. Home Assistant users can override these built-in names.

**Comment by JeffResc** [comment]
Thanks, Jeff. Since this is just an arbitrary string, is there any technical reasons that two strings can’t be the same? I understand that two variables cannot be the same for obvious reasons, but strings should be fine? Please let me know.

**Comment by JeffLIrion** [comment]
https://github.com/home-assistant/core/blob/183f94364a3c0fe19575f7b4d03ee30fbe4d9070/homeassistant/components/androidtv/media_player.py#L410-L412

If there are two apps named "YouTube" and a user selects the "YouTube" source, it's not clear which app will be launched.

**Comment by JeffResc** [comment]
@JeffLIrion I addressed the test errors if you would like to go ahead and take another look at this.

**Comment by JeffLIrion** [comment]
Thanks!

**Pull request merged**